Functions.Templates/Templates/EventHubTrigger-JavaScript/function.json

```diff
@@ -7,7 +7,8 @@
         "eventHubName": "samples-workitems",
         "connection": "",
         "cardinality": "many",
-        "consumerGroup": "$Default"
+        "consumerGroup": "$Default",
+        "dataType": "string"

```

Comment:
Is this a new property, if so can you add it
https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Bindings/bindings.json#L806